android/build.gradle

```diff
@@ -49,4 +49,5 @@ android {
 
 dependencies {
     implementation 'io.piano:analytics:3.2.1'
+    //implementation 'io.piano.android:analytics:3.4.0'
```

Comment:
Maybe add a comment to indicate a major impact ?

Reply:
Hi, I wanted to keep track of the future evolution of the plugin (1.0.0). Besides, I'm going to create a ticket with this in mind.

Reply:
Here is the ticket for next evol: https://github.com/alex596/piano_analytics_plugin/issues/10